vassal-app/src/main/java/VASSAL/chat/ServerAddressBook.java

```diff
@@ -77,8 +77,6 @@
   protected static final String P2P_TYPE = P2PClientFactory.P2P_TYPE;
   protected static final String P2P_MODE_KEY = P2PClientFactory.P2P_MODE_KEY;
   protected static final String P2P_SERVER_MODE = P2PClientFactory.P2P_SERVER_MODE;
-  protected static final String P2P_CLIENT_MODE = P2PClientFactory.P2P_CLIENT_MODE;
```

Comment:
Why is this being removed?

Reply:
It is not used anywhere

Reply:
I put it back with a deprecated marker, if modules extend this class they can define that field themselves.